nuc_morph_analysis/analyses/dataset_images_for_figures/figure_helper.py

```diff
@@ -317,7 +369,7 @@ def draw_scale_bar_matplotlib(
 
     # the height is 5% of the axes height
 
-    ax.plot([x, x2], [y, y], color="w", linewidth=2)
+    ax.plot([x, x2], [y + 3, y + 3], color="w", linewidth=2)
```

Comment:
why was this change made? Won't it affect other panels in Figure1

Reply:
Ok. I see now that this is just the scale bar location which is not critically important. I'll approve :)

Reply:
Yes for me this was placing the scale bar with the micron symbol and the scalebar touching/overlapping and I wanted a physical space between them